src/api/Makefile.am

```diff
@@ -0,0 +1,17 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we should continue the pattern of using includes for new Makefile (and keep working on fixing the existing ones). As an example, see how it's done in the plugins/ tree. This still preserves the Makefile modularity (one Makefile per "level"), but allows for much better parallelism.